.secrets.baseline

```diff
@@ -62,7 +62,7 @@
       {
         "hashed_secret": "8c0fdc72bf4daf2a2338287334d0cbd221fa9ef4",
         "is_verified": false,
-        "line_number": 28,
+        "line_number": 30,
```

Comment:
the secrets baseline is outdated, since you deleted `.travis.yml` it shouldn't be in here anymore

Reply:
thanks pauline!